run/GCClassic/input_options.yml.templates/input_options.yml.fullchem

```diff
@@ -0,0 +1,426 @@
+aerosols:
+
+  carbon:
+    activate: true
+    brown_carbon: false
+    enhance_black_carbon_absorption:
+      activate: true
+      hydrophilic: 1.5
+      hydrophobic: 1.0
+
+  complex_SOA:
+    activate:  ${RUNDIR_COMPLEX_SOA}
+    semivolatile_POA: ${RUNDIR_SVPOA}
+
+  dust:
+    activate: true
+    acid_uptake_on_dust: ${RUNDIR_ACID_UPTAKE}
+
+  sea_salt:
+    activate: true
+    SALA_radius_bin_in_um: [0.01, 0.5]
+    SALC_radius_bin_in_um: [0.5,  8.0]
+    marine_organic_aerosols: ${RUNDIR_MARINE_POA}
+
+  stratosphere:
+    settle_strat_aerosol: true
+    polar_strat_clouds:
+      activate: true
+      het_chem: true
+    allow_homogeneous_NAT: false
+    NAT_supercooling_req_in_K: 3.0
+    supersat_factor_req_for_ice_nucl: 1.2
+    calc_strat_aero_optdepth: true
+
+  sulfate:
+    activate: true
+    metal_cat_SO2_oxidation: true
+
+#============================================================================
+# Settings for diagnostics (other than HISTORY and HEMCO)
+#============================================================================
+extra_diagnostics:
+
+  obspack:
+    activate: false
+    quiet_logfile_output: false
+    input_file: ./obspack_co2_1_OCO2MIP_2018-11-28.YYYYMMDD.nc
+    output_file: ./OutputDir/GEOSChem.ObsPack.YYYYMMDD_hhmmz.nc4
+    output_species:
+      - CO
+      - 'NO'
+      - O3
+
+  planeflight:
+    activate: false
+    flight_track_file: Planeflight.dat.YYYYMMDD
+    output_file: plane.log.YYYYMMDD
+
+  legacy_bpch:                #          1         2         3
+     output_menu:             # 1234567890123456789012345678901
+       schedule_output_for_JAN: 3000000000000000000000000000000
+       schedule_output_for_FEB: 30000000000000000000000000000
+       schedule_output_for_MAR: 3000000000000000000000000000000
+       schedule_output_for_APR: 300000000000000000000000000000
+       schedule_output_for_MAY: 3000000000000000000000000000000
+       schedule_output_for_JUN: 300000000000000000000000000000
+       schedule_output_for_JUL: 3000000000000000000000000000000
+       schedule_output_for_AUG: 3000000000000000000000000000000
+       schedule_output_for_SEP: 300000000000000000000000000000
+       schedule_output_for_OCT: 3000000000000000000000000000000
+       schedule_output_for_NOV: 300000000000000000000000000000
+       schedule_output_for_DEC: 3000000000000000000000000000000
+    gamap:
+      diaginfo_dat_file: ./diaginfo.dat
+      tracerinfo_dat_file: ./tracerinfo.dat
+    bpch_diagnostics:
+      TOMAS_aerosol_emissions: "0 all"
+      TOMAS_rate: "0 all"
+      TOMAS_3D_rate: "0 all"
+# NOTE: These might be obsolete in 14.0.0, so comment out
+#    ND51_satellite:
+#      activate: false
+#      output_file: ts_satellite.YYYYMMDD.bpch
+#      tracers: [1, 2, 501]
+#      UTC_hour_for_write: 1
+#      averaging_period_in_LT: [13, 15]
+#      IMIN_and_IMAX_of_region: [1, 72]
+#      JMIN_and_JMAX_of_region: [1, 46]
+#      LMIN_and_LMAX_of_region: [1, 1]
+#
+#    ND51b_satellite:
```

Comment:
Could this be more descriptive of how ND51b differs from ND51?

Reply:
We can.  But if we get the PR #1134 into 14.0.0, that will make these obsolete.